src/pages/nfts/[address].tsx

```diff
@@ -70,7 +70,7 @@ const GET_NFT = gql`
     nft(address: $address) {
           name
           address
-          image
+          image(width: 800)
```

Comment:
I got this at 1400 in next PR opening. The max width is 750 so using double for retina displays ie 1400.